ci/scripts/test-airgapped-build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+# Prefetch bazel airgapped dependencies.
+util/prep-bazel-airgapped-build.sh -f
```

Comment:
Just a heads up that this step can be flaky. We never determined the root cause for it, but now that we're making this blocking, we may want to keep an eye out for PRs failing this step.

Reply:
#15176 attempts to fix this. I will wait to merge this until after #15176 is merged.